src/Grpc.Net.Client/Balancer/BalancerAddress.cs

```diff
@@ -0,0 +1,71 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
```

Comment:
```suggestion
// Copyright 2021 The gRPC Authors
```

Do we need to do something to updates the copyright dates in the other files? Also, why individual headers in each source file instead of a top-level one?

Reply:
I looked at other grpc repos and the standard is to leave it on the date the repo started.